examples/src/test/java/io/numaproj/numaflow/examples/utils/TestDatum.java

```diff
@@ -0,0 +1,35 @@
+package io.numaproj.numaflow.examples.utils;
+
+import io.numaproj.numaflow.mapper.Datum;
+import lombok.Builder;
+
+import java.time.Instant;
+import java.util.Map;
+
+@Builder
+public class TestDatum implements Datum, io.numaproj.numaflow.sourcetransformer.Datum {
```

Comment:
This is just to demonstrate how we can use a concrete class to replace mocking interfaces in unit tests. For this to be available to numaflow users, we need to move it from the examples pkg to the outer src pkg.  WDYT? @yhl25

Reply:
Creating these classes is a simple task for users. Our provided examples should be enough, we don't need to include them in our primary code.